src/Entities/Schema.php

```diff
@@ -177,9 +177,24 @@ public function __construct()
             'sessions/{sessionId}' => function (array $content) {
                 return new Session($content);
             },
-            'login' => function (array $content) {
+            'users/login' => function (array $content) {
                 return new Session($content);
             },
+            'users/signup' => function (array $content) {
+                return new User($content);
+            },
+            'users/forgot-password' => function (array $content) {
```

Comment:
I.e. this factories need only to build object from json response for `GET` and `PUT`